includes/html/pages/addhost.inc.php

```diff
@@ -254,8 +260,12 @@
             <label for="cryptoalgo" class="col-sm-3 control-label">Crypto Algorithm</label>
             <div class="col-sm-9">
               <select name="cryptoalgo" id="cryptoalgo" class="form-control input-sm">
-                <option value="AES" selected>AES</option>
-                <option value="DES">DES</option>
+                <option value="DES" selected>DES</option>
```

Comment:
intended change setting DES as default?

Reply:
OCD about how it will look. AES will be the one with "more options"? If that makes sense?